wear/src/main/java/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -235,26 +244,72 @@ class HomeActivity : ComponentActivity(), HomeView {
                 }
             }
 
-        Chip(
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(top = if (index == 0) 30.dp else 10.dp),
-            icon = {
-                if (iconBitmap != null) {
-                    Image(asset = iconBitmap)
-                } else
-                    Image(asset = CommunityMaterial.Icon.cmd_cellphone)
-            },
-            label = {
-                Text(
-                    text = attributes["friendly_name"].toString(),
-                    maxLines = 2,
-                    overflow = TextOverflow.Ellipsis
+        if (entity.entityId.split(".")[0] in HomePresenterImpl.toggleDomains) {
+            ToggleChip(
+                checked = entity.state == "on",
+                onCheckedChange = {
+                    presenter.onEntityClicked(entity)
+                    mainScope.launch {
+                        entityViewModel.entitiesResponse = presenter.getEntities()
+                    }
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = if (index == 0) 30.dp else 10.dp),
+                appIcon = {
+                    if (iconBitmap != null) {
+                        Image(asset = iconBitmap)
+                    } else
+                        Image(asset = CommunityMaterial.Icon.cmd_cellphone)
+                },
```

Comment:
```suggestion
                appIcon = {
                    Image(asset = iconBitmap ?: CommunityMaterial.Icon.cmd_cellphone)
                },
```

Reply:
Thank you for the suggestion! As I had other changes it was easier to incorporate in the same commit :)